dubbo-common/src/main/java/org/apache/dubbo/common/utils/StringUtils.java

```diff
@@ -530,19 +532,31 @@ public static boolean isEquals(String s1, String s2) {
     }
 
     /**
-     * is integer string.
+     * is positive integer or zero string.
```

Comment:
It may be more appropriate to change to the following expression: “determine whether it is a positive integer or zero string”

Reply:
i think this method need change method name or add some comment, but i don't know what to do, other people invoke this method must be careful.

Reply:
@levis9527 
You can change the description to: determine whether it is a positive integer or zero string